### ISSUE TITLE
MongoDB Timestamp type is not supported

### ISSUE THREAD

**Issue by ghost** [mrsarm/mongotail#29]
If a log entry is written where a MongoDB timestamp is included, the logger fails with an exception:

```
Traceback (most recent call last):
  File "/usr/local/bin/mongotail", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/mongotail/mongotail.py", line 238, in main
    tail(client, db, args.n, args.follow, args.verbose, args.metadata)
  File "/usr/local/lib/python2.7/site-packages/mongotail/mongotail.py", line 83, in tail
    print_obj(result, verbose, metadata, server_version)
  File "/usr/local/lib/python2.7/site-packages/mongotail/out.py", line 196, in print_obj
    warn('Unknown registry\nDump: %s' % json_encoder.encode(obj))
  File "/usr/local/lib/python2.7/site-packages/mongotail/jsondec.py", line 60, in encode
    result = super(JSONEncoder, self).encode(o)
  File "/usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/lib/python2.7/site-packages/mongotail/jsondec.py", line 57, in default
    return json.JSONEncoder.default(self, o)
  File "/usr/local/Cellar/python@2/2.7.15_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: Timestamp(1548943026, 1) is not JSON serializable
```

I'm using:

```
mongotail --version
mongotail 2.2.0 <https://github.com/mrsarm/mongotail> (python 2.7.15)
```

**Comment by mrsarm** [comment]
Hi,

Thanks for reporting the error. Actually, I didn't know about this type, now I see in the documentation a reference about it: https://docs.mongodb.com/manual/reference/bson-types/#timestamps

I'm working in my spare time in a new release of MongoDB with a few changes, so I'll add support for this object type too.

Meanwhile you can execute mongotail with the `-v` option, it's not ideal, but it allows to print the logs without the formatting that it's causing the error, should work, let me know if not.

**Comment by mrsarm** [comment]
Hey @maitscha ,

I made the patch to add support to the `Timestamp` type, but I didn't released a new version yet because I going to add one more change these days. Could you try the master version?

First remove the installed version with:

    $ sudo pip uninstall mongotail

But then I recommend you to install it with Python 3 instead of Python 2 (you need pip3):

    $ sudo pip3 install https://github.com/mrsarm/mongotail/archive/master.zip

If you don't have Python3 nor pip3, just change the `pip3` command by `pip` (Python 2). I have to add the Python 3 recommendation in the README file, the serialization for some types works best in Python 3.

**Comment by mrsarm** [comment]
Released in version [2.3.0](https://github.com/mrsarm/mongotail/releases/tag/2.3.0)
